de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Average_Product_Prices.sql

```diff
@@ -1,28 +1,12 @@
-DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.average_product_prices ( IN p_datefrom DATE, IN p_dateto DATE, IN p_issotrx character(1) );
-CREATE OR REPLACE FUNCTION de_metas_endcustomer_fresh_reports.average_product_prices ( IN p_datefrom DATE, IN p_dateto DATE, IN p_issotrx character(1) )
-RETURNS TABLE 
-(
-	ProduktNr character varying(40),
-	ProduktName character varying(225),
-	Merkmal text,
-	Menge numeric,
-	Menge_Lieferung numeric,
-	Mengenenheit character varying,
-	Preis numeric,
-	BetragCHF text,
-	Wahrung character(3),
-	Preisenheit character varying,
-	ProduktKategorie character varying(60),
-	GeschaftspartnerNr character varying(40),
-	GeschaftspartnerName character varying(60),
-	Country character varying,
-	ADR character varying,
-	Label character varying,
-	BelegNr character varying(30), 
-	BewegungsDatum timestamp without time zone
-)
-AS
-$$
+DROP FUNCTION de_metas_endcustomer_fresh_reports.average_product_prices(date, date, character);
+
+CREATE OR REPLACE FUNCTION de_metas_endcustomer_fresh_reports.average_product_prices(
+    IN p_datefrom date,
+    IN p_dateto date,
+    IN p_issotrx character)
+  RETURNS TABLE(produktnr character varying, produktname character varying, merkmal text, menge numeric, menge_lieferung numeric, mengenenheit character varying, preis numeric, betragchf text, wahrung character, preisenheit character varying, produktkategorie character varying, geschaftspartnernr character varying, geschaftspartnername character varying, country character varying, adr character varying, label character varying, belegnr character varying, bewegungsdatum timestamp without time zone) AS
```

Comment:
please retain this formatting; otherwise we can'T see which columns changed

Reply:
Done